crates/miden-objects/src/lib.rs

```diff
@@ -11,14 +11,15 @@ pub mod address;
 pub mod asset;
 pub mod batch;
 pub mod block;
+pub mod errors;
 pub mod note;
+pub mod protocol;
```

Comment:
Do we need to make the `protocol` module public? We could probably just do something like:
```rust
mod protocol;
pub use protocol::ProtocolLib`;
```
right?